Project.toml

```
@@ -14,6 +14,7 @@ RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
 
 [compat]
 julia = "1"
+GeoInterface = "^1"
```

I had to look it up in https://pkgdocs.julialang.org/v1/compatibility/, but the caret is the default. Let's leave it out since that's commonly done.

```suggestion
GeoInterface = "1"
```

---

My preference is always explicit over implicit... but I trust you judgment over mine